arch/csr/mseccfg.yaml

```diff
@@ -6,9 +6,110 @@ name: mseccfg
 long_name: Machine Security Configuration
 address: 0x747
 priv_mode: M
-length: 64
-description: Machine Security Configuration
+length: MXLEN
+description: Machine Security Configuration register is used for configuring various security mechanisms present on the hart and only accessible in Machine mode.
```

Comment:
Remove "and only accessible in Machine Mode", since it's redundant with the "priv_mode", above.